lib/Pipfile

```diff
@@ -29,6 +29,7 @@ parameterized = "*"
 psycopg2-binary = "*"
 pydot = "*"
 tensorflow = {version = ">=2.0.0", markers = "python_version < '3.8'"}
+torchvision = "*"
```

Comment:
You need to review my PR #1407 , since this part of the Pipfile is changed there — and my PR came first by several days :smile: 

One you do, please put this `test-requirements.txt`